theano/tensor/tests/test_basic.py

```diff
@@ -1577,10 +1577,6 @@ def test_py_c_match():
     integers=(randint(2, 3), randint(2, 3)),
     int8=[numpy.arange(-127, 128, dtype='int8'),
           numpy.arange(-127, 128, dtype='int8')[:, numpy.newaxis]],
-    uint8=[numpy.arange(0, 255, dtype='uint8'),
-           numpy.arange(0, 255, dtype='uint8')[:, numpy.newaxis]],
-    uint16=[numpy.arange(0, 65535, dtype='uint16'),
-            numpy.arange(0, 65535, dtype='uint16')[:, numpy.newaxis]],
```

Comment:
About the uint8, ideally, we should fix Theano for it, but if we make uint16 work, we could make a new issue about making arctan2 work with uint8/uint8.